### PR TITLE
updated documentation about envelope initiation and invitation

### PR DESCRIPTION
**Pull request by OSoloviova**
Expanded and updated the documentation on envelope initiation and invitations, replaced and added screenshots.

### CODE REVIEW

File: docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/sendEnvelopeAsGuest/sendEnvelopeAsGuest.rst [hunk 1]
```diff
@@ -7,18 +7,46 @@ User is able to send envelope without authorisation. To use this functionality u
...
+4. Follow the link without authorisation
+5. Fill in all required fields assigned to the sender role and roles that inherit the sender role.
+6. Send the envelope and follow the instructions
 
 How to create share template link?
 ==================================
 
 1. Create a template
-2. Fill in envelope subject and fill in all roles in the flow with mailboxes
+2. Fill in envelope subject and fill in all roles in the flow with mailboxes or use Same as functionality
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/sendEnvelopeAsGuest/sendEnvelopeAsGuest.rst:49]: Same as -> "Same as" or same as

File: docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst [hunk 2]
```diff
@@ -41,11 +47,80 @@ We're able to invite to an envelope such as customers of the platform and non-re
...
+.. image:: pic_inviteToEnvelopeByEmail/inviteSuccessPage.png
+   :width: 400
+   :align: center
+
+
+How to accept invitation and send envelope without authorisation for registered users?
+======================================================================================
+
+1. To accept the invitation to the envelope, the user can follow the link sent to their email address. 
+Enter valid credentials and sign in. Authorization through third-party services is also available.
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst:97]: please unify Authorization and Authorisation. Chose one pattern of the word

**Review by kolapse2 — CHANGES_REQUESTED** [review]
please make fixes described in comment

File: docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst [hunk 3]
```diff
@@ -2,32 +2,38 @@
...
 Invite participant to envelope by email from the draft page
 ===========================================================
 
-* Open draft of the envelope without predefined processing flow and participant there
-* To the input of the particular role enter the participant email address (you're able to left the email address and don't choose the mailbox from drop-down list)
-* After completion of the envelope configuration you're able to send envelope
-* At the modal window you will see the alert message which says that you try to send envelope to email
-* Just confirm and envelope will be send
+* Open draft of the envelope without predefined processing flow and participant there. 
+You can also use a draft created based on the template where the "Same as" function was used during the flow setup.
```
[the remark this answers is not in the document]
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst]: done

File: docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst [hunk 4]
```diff
@@ -41,11 +47,80 @@ We're able to invite to an envelope such as customers of the platform and non-re
...
+   :align: center
+
+4. Click on "Send" button
+5. If there are roles in the envelope flow that inherit from the current role, then the envelope with the functionality of the next role (Approver/Signer) will be displayed.
+
+.. image:: pic_inviteToEnvelopeByEmail/inviteSameAsRoles.png
+   :width: 400
+   :align: center
+
+6. Perform the necessary actions depending on the role funcrionality and then complete (send) the envelope.
```
[the remark this answers is not in the document]
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst]: done

File: docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst [hunk 5]
```diff
@@ -41,11 +47,80 @@ We're able to invite to an envelope such as customers of the platform and non-re
 Auto Deposit Function
```
[the remark this answers is not in the document]
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst]: done

File: docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/sendEnvelopeAsGuest/sendEnvelopeAsGuest.rst (hunk 1 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/sendEnvelopeAsGuest/sendEnvelopeAsGuest.rst]: done

File: docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst (hunk 2 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst:97]: done

File: docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst [hunk 6]
```diff
@@ -38,14 +44,83 @@ We're able to invite to an envelope such as customers of the platform and non-re
 * If you leave the email address at the input at the processing flow, participant will be invited by email
 * If you select the mailbox from the drop-down list at the processing flow, participant will get envelope as usual
 
-Auto Deposit Function
+"Default mailbox" function
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst:47]: ======== - should be equal to length of the title

**Review by kolapse2 — CHANGES_REQUESTED** [review]
need one more small fix

**Review by kolapse2 — APPROVED** [review]
approved

**Pull request merged**